c++/src/capnp/rpc.c++

```diff
@@ -2052,7 +2121,7 @@ private:
                    kj::Array<kj::Maybe<kj::Own<ClientHook>>> capTableArray,
                    const AnyPointer::Reader& params,
                    bool redirectResults, kj::Own<kj::PromiseFulfiller<void>>&& cancelFulfiller,
-                   uint64_t interfaceId, uint16_t methodId)
+                   uint64_t interfaceId, uint16_t methodId, bool realtime = false)
```

Comment:
This is probably not the most elegant solution, but my alternative was to refactor the `Context` to have e.g. `RpcCallContext` and `RpcRealtimeCallContext`. But I wanted to keep the changes in this PR minimal, and before I do the bigger change I would like to have the opinion from a maintainer :blush: